configs: Add replacement policy options for GPUFS

abmerop opened the pull request:
GPU_VIPER.py was modified to use these options but they did not exist, breaking GPUFS.  This commit adds them to fix the issue.

Change-Id: I0095f400ea606c4e8d91a41870ef208465cef803

mattsinc reviewed (approved):
@yuxiaojia : FYI

mattsinc reviewed (commented):
@abmerop : while we're at it, do we want to update the way that GPUFS picks the replacement policy?  Or I guess @yuxiaojia could make a separate commit for that?

abmerop commented:
> @abmerop : while we're at it, do we want to update the way that GPUFS picks the replacement policy? Or I guess @yuxiaojia could make a separate commit for that?

Currently both GPUFS and GPU-SE are using the common ruby config to set the replacement policy, is modifying that what you were thinking?

abmerop committed:
configs: Add replacement policy options for GPUFS

GPU_VIPER.py was modified to use these options but they did not exist,
breaking GPUFS.  This commit adds them to fix the issue.

Change-Id: I0095f400ea606c4e8d91a41870ef208465cef803

mattsinc commented:
> > @abmerop : while we're at it, do we want to update the way that GPUFS picks the replacement policy? Or I guess @yuxiaojia could make a separate commit for that?
> 
> Currently both GPUFS and GPU-SE are using the common ruby config to set the replacement policy, is modifying that what you were thinking?

I think?  I think you're saying by adding these flags here, a user can pick other replacement policies now in both GPUFS and GPUSE?

abmerop commented:
> I think? I think you're saying by adding these flags here, a user can pick other replacement policies now in both GPUFS and GPUSE?

Yes, they can do that now, and the command line parameter will be the same.
